Admin/PostAdmin.php

```diff
@@ -32,7 +31,7 @@ class PostAdmin extends Admin
     protected $userManager;
 
     /**
-     * @var Pool
+     * @var FormatterPool
```

Comment:
`Pool` was used. I suggest to remove the *real* unused statement instead: `FormatterPool`.